diff -up st/config.def.h newst/config.def.h
--- st/config.def.h	2020-09-21 22:47:05.603736688 +0800
+++ newst/config.def.h	2020-09-21 22:44:34.912367000 +0800
@@ -5,7 +5,7 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+static char *font = "Noto Sans Mono CJK SC:pixelsize=24:antialias=true:autohint=true";
 static int borderpx = 2;

 /*
